annotate.el

```diff
@@ -5,7 +5,7 @@
 ;; Maintainer: Bastian Bechtold
 ;; URL: https://github.com/bastibe/annotate.el
 ;; Created: 2015-06-10
-;; Version: 0.4.7
+;; Version: 0.4.8
```

Comment:
As this is more than a bugfix, it should bump to 0.5.0 (first number for breaking changes, second number for new features, third number for bug fixes).